oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -27,51 +27,57 @@ class ComputerSystem(RedfishJsonValidator):
 
     SCHEMA_NAME = 'ComputerSystem'
 
-    def __init__(self, sh_dict, sht_dict):
+    def __init__(self, server_hadware, server_hadware_types):
```

Comment:
There is a typo in this parameters, the correct is **hardware** not **hadware**.